app/src/main/java/com/rarilabs/rarime/api/ext_integrator/ext_int_action_preview/handlers/ext_int_query_proof_handler/ExtIntQueryProofHandler.kt

```diff
@@ -82,7 +87,7 @@ fun ExtIntQueryProofHandler(
                     message = context.getString(R.string.query_proof_success_subtitle),
                 )
             )
-            onSuccess.invoke()
+            onSuccess.invoke(redirectURL)
```

Comment:
we use `destination`, `redirect_uri` and `redirectURL` for the same param
let's stick to `redirectURI`

Reply:
done